Makefile

```diff
@@ -82,7 +82,7 @@ build:
 	go build -ldflags "-X main.GitTag=$(GIT_TAG) -X main.GitHash=$(GIT_HASH) -s -w" ./cmd/telliot
 
 .PHONY: generate-config-docs
-generate-config-docs:
+generate-config-docs: ## Use the example env file and the actual golang config struct to generate the setup doc page. .... <- add more info how it works.
```

Comment:
I meant this only as an example :)  Please make it a proper full comment.

Reply:
Done!